server/model/Publicacao.js

```diff
@@ -0,0 +1,96 @@
+import mongoose from 'mongoose';
+import * as erro from '../util/ErroHandler';
+import { UsuarioService }  from '../service/UsuarioService';
+
+const Schema = mongoose.Schema;
+
+const self = this;
+
+/**
+ * Schema que representa o Usuario no banco de dados
+ * validações adicionadas para manter a consistencia dos dados
+ * que iremos guardar.
+ *
+ * @author Gustavo Oliveira
+ */
+const publicacaoSchema = new Schema({
+  ownerUsername : {
+    type: String,
+    required: [true, erro.CADASTRO.VALIDACAO_OWNER]
+  },
+
+  url : {
+    type: String,
+    required: [true, erro.CADASTRO.VALIDACAO_URL]
+  },
+
+  description : String,
+
+  genres: [String],
+
+  aproves: [String]
```

Comment:
O que seria o aproves? Se for algo relacionado a like and deslike, melhor deixar em variaveis separadas.